src/main/java/org/openrewrite/java/testing/junit5/MockitoJUnitAddMockitoSettingsLenientStrictness.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2021 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.testing.junit5;
+
+import org.openrewrite.ExecutionContext;
+import org.openrewrite.Preconditions;
+import org.openrewrite.Recipe;
+import org.openrewrite.TreeVisitor;
+import org.openrewrite.java.JavaIsoVisitor;
+import org.openrewrite.java.search.UsesType;
+import org.openrewrite.java.tree.J;
+
+public class MockitoJUnitAddMockitoSettingsLenientStrictness extends Recipe {
```

Comment:
As indicated I wonder if we should make this [part of `MockitoJUnitToMockitoExtension`](https://github.com/openrewrite/rewrite-testing-frameworks/blob/efcc1e4904e90d0c7a1cbd6b1a230ff698a3917d/src/main/java/org/openrewrite/java/testing/junit5/MockitoJUnitToMockitoExtension.java#L53) instead of having a separate recipe. That'll depend on whether Mockito 1 and 2 could be used on JUnit 4 without the use of that test MockitoRule or MockitoTestRule.